Spokestack/TextToSpeech.swift

```diff
@@ -81,28 +89,37 @@ import AVFoundation
     /// - Note: The URL will be invalidated within 60 seconds of generation.
     /// - Parameter input: Parameters that specify the speech to synthesize.
     @objc public func synthesize(_ input: TextToSpeechInput) -> Void {
-        self.synthesize(input: input, success: successHandler)
+        self.synthesize(input: input, success: successHandler(result:))
     }
     
-    private func synthesize(input: TextToSpeechInput, success: ((URL) -> Void)?) {
+    private func synthesize(input: TextToSpeechInput, success: ((TextToSpeechResult) -> Void)?) {
         let session = URLSession(configuration: URLSessionConfiguration.default)
-        var request = URLRequest(url: URL(string: "https://core.pylon.com/speech/v1/tts/synthesize")!)
-        request.addValue(self.configuration.authorization, forHTTPHeaderField: "Authorization")
+        var request = URLRequest(url: URL(string: "https://api.spokestack.io/v1")!)
         request.addValue("application/json", forHTTPHeaderField: "Content-Type")
+        let xRequestID = "x-request-id"
+        request.addValue(input.id, forHTTPHeaderField: xRequestID)
         request.httpMethod = "POST"
-        var inputFormat: String
-        switch input.inputFormat {
-        case .text:
-            inputFormat = "text"
-            break
-        case .ssml:
-            inputFormat = "ssml"
-            break
+        let body: [String:Any] = [
+            "query":"query synthesis($voice: String!, $ssml: String!) {synthesize(voice: $voice, ssml: $ssml) {url}}",
```

Comment:
@NeilMenne asked me to have a platform-specific name for my GraphQL query to ease auditing. Do you want them to be parallel, Neil, or just have Android queries call out their platform?

Reply:
I can do whatever for the query name.